perfkitbenchmarker/providers/openstack/os_virtual_machine.py

```diff
@@ -33,7 +33,7 @@ class OpenStackVirtualMachine(virtual_machine.BaseVirtualMachine):
     """Object representing an OpenStack Virtual Machine"""
 
     CLOUD = 'OpenStack'
-    DEFAULT_USERNAME = 'ubuntu'
+    DEFAULT_USERNAME = FLAGS.openstack_image_username
```

Comment:
DEFAULT_USERNAME is no longer a constant, nor is its name accurate anymore. I suggest removing it and initializing self.user_name from the flag in the __init__ method.